_episodes/interviews/2023-05-26-rainer-stropek.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "CoderDojo and Rust Linz with Rainer Stropek"
+date: 2023-05-26T15:22:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-05-26-rainer-stropek.mp3
+duration: "55:24"
+length: "53188650"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Allen Wyma and [Zeeshan Ali Khan](https://toot.cat/@zeenix) talk with [Rainer Stropek](https://github.com/rstropek), creator of [CoderDojo Linz](https://linz.coderdojo.net/) and [Rust Linz](https://rust-linz.at/).
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+
+- [@0:00] - Introduction to Rust Linz
+- [@3:23] - How did Rainer get into teaching kids in Austria how to code
+- [@5:17] - CoderDojo's vision in creating a coding club
```

Comment:
I'm a little on the fence about this episode in terms of its connection to Rust. There's some Rust in here, which is good, but otherwise it's a more general discussion about coding clubs + about this specific one. In this instance I think it's okay because teaching kids to code is a topic that's near and dear to my heart, though just a heads up that we should try to keep things focused on Rust rather than programming as a more general topic.

cc @bstrie